madara/crates/primitives/chain_config/src/chain_config.rs

```diff
@@ -370,6 +360,7 @@ impl ChainConfig {
                 strk_fee_token_address: self.native_fee_token_address,
                 eth_fee_token_address: self.parent_fee_token_address,
             },
+            is_l3: true,
```

Comment:
shall we keep it false by default?

```suggestion
            is_l3: false,
```

Reply:
done